File: Codechef/Speciality/Speciality.cpp
#include <iostream>
using namespace std;

int main()
{
    int T;
    cin >> T;
    while (T--)
    {
        int X, Y, Z;
        if (X > Y)
        {
            if (X > Z)
            {
                cout << "Setter" << endl;
            }
            else
            {
                cout << "Editorialist" << endl;
            }
        }
        else
        {
            if (Y > Z)
            {
                cout << "Tester" << endl;
            }
            else
            {
                cout << "Editorialist" << endl;
            }
        }
    }
    return 0;
}

File: Codechef/Speciality/Speciality.py
T = int(input())
while T:
    T -= 1
    X, Y, Z = map(int, input().split())
    if X > Y:
        if X > Z:
            print("Setter")
        else:
            print("Editorialist")
    else:
        if Y > Z:
            print("Tester")
        else:
            print("Editorialist")
